test/e2e/serial/tests/non_regression_fundamentals.go

```diff
@@ -149,4 +153,211 @@ var _ = Describe("[serial][fundamentals][scheduler][nonreg] numaresources fundam
 			Expect(schedOK).To(BeTrue(), "pod %s/%s not scheduled with expected scheduler %s", updatedPod.Namespace, updatedPod.Name, serialconfig.Config.SchedulerName)
 		})
 	})
+
+	Context("using the NUMA-aware scheduler with NRT data", func() {
+		var cpusPerPod int64 = 2 // must be even. Must be >= 2
+
+		DescribeTable("[node1] against a single node",
+			// the ourpose of this test is to send a burst of pods towards a node. Each pod must require resources in such a way
```

Comment:
a small non-blocking nit:
```suggestion
			// the purpose of this test is to send a burst of pods towards a node. Each pod must require resources in such a way
```

Reply:
thanks, I'll queue for the next batch of cleanups (which we should do regularly anyway)

Reply:
https://github.com/openshift-kni/numaresources-operator/issues/495